newsletter-march-2022.rst

```diff
@@ -0,0 +1,63 @@
+.. post:: Mar 10, 2022
+   :tags: newsletter, python
+   :author: Eric Holscher
+   :location: BND
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - March 2022
+=====================================
+
+It's been pretty quiet on the company front in February,
+with nothing much to report.
+**We're actively working on our latest job description,
+which will be a product-focused Python development position.**
+f you're interested, please `let us know`_.
+
+.. _let us know: mailto:hello@readthedocs.org?subject=Job%20Posting
+
+New features
+------------
+
+In February we continued to work on refactors and internal changes.
+Among the major user-facing changes:
+
+- **We now have the CDN on Read the Docs for Business in beta**, with a couple accounts testing it. If you're interested, please `contact us`_.
+- You can now `cancel builds <https://github.com/readthedocs/readthedocs.org/pull/8850>`_ via a button in the dashboard. This will save resources, and allow you to ensure that you don't hit concurrency limits if you trigger a few builds at once.
```

Comment:
The next step in this direction is https://github.com/readthedocs/readthedocs.org/issues/8961. I'm not sure if it is worth mentioning it here as well or not.